config/rbac/role.yaml

```diff
@@ -15,6 +15,13 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - ""
+  resources:
+  - endpoints
```

Comment:
I recall a conversation on Slack about not needing to query endpoints directly. Instead use the service DNS name and the client should resolve all the IPs from that.
Is that an option?

Reply:
Yeah, I've removed this permission.